components/clientComponents/forms/Form/Form.tsx

```diff
@@ -44,15 +44,28 @@ const SubmitButton: React.FC<SubmitButtonProps> = ({
   const [submitTooEarly, setSubmitTooEarly] = useState(false);
   const screenReaderRemainingTime = useRef(formTimerState.remainingTime);
 
+  // TODO: may want to consider a maximum delay
+
   // calculate initial delay for submit timer
   const secondsBaseDelay = 2;
   const secondsPerFormElement = 2;
-  const submitDelaySeconds = secondsBaseDelay + numberOfRequiredQuestions * secondsPerFormElement;
 
   const formTimerEnabled = process.env.NEXT_PUBLIC_APP_ENV !== "test";
 
   // If the timer hasn't started yet, start the timer
-  if (!formTimerState.timerDelay && formTimerEnabled) startTimer(submitDelaySeconds);
+  if (!formTimerState.timerDelay && formTimerEnabled) {
+    // Calculated here to avoid being called on each checkTimer interval (every second)
+    const submitDelaySeconds =
```

Comment:
Wondering if this work should happen outside of the actual button ... pass the delay as a prop to the button.

Thoughts?

Reply:
Yup that could easily pulled out into a function and the result of that function passed into the Submit Button